src/api/index.js

```diff
@@ -23,26 +23,30 @@ export function getHosts() {
   return doGet('/api/inventory/v1/hosts');
 }
 
-export function downloadPlaybook(selectedIds) {
-  return new Promise((resolve) => {
-    const tab = downloadPlaybooksParamCreator({
-      selectedRemediations: selectedIds,
-      options: { baseURL: API_BASE },
-    }).then((res) => {
-      const { search, pathname } = res.urlObj;
-      const url = `${API_BASE}${pathname}${search}`;
-      window.location.assign(url);
-      return true;
-    });
-
-    const handle = setInterval(async () => {
-      if (tab) {
-        clearInterval(handle);
-        resolve();
-      }
-    }, 500);
-  });
-}
+export const downloadPlaybook = async (selectedIds) => {
+  try {
+    if (selectedIds.length > 1) {
+      downloadPlaybooksParamCreator({
```

Comment:
@bastilian
The OpenAPI spec defines downloadPlaybooks as a GET, so the client hard-codes a GET withthe query params. this only “works” when you pass a single ID. As soon as you send multiple IDs, the generated query‐param format trips the endpoint (which in reality expects a POST with a JSON body).